Content.Server/Power/EntitySystems/ChargerSystem.cs

```diff
@@ -34,30 +34,27 @@ public override void Update(float frameTime)
     {
         foreach (var comp in EntityManager.EntityQuery<ChargerComponent>())
         {
```

Comment:
This should only be iterating chargers with stuff in them.

Reply:
I think I'm already doing that, the git diff is just making it hard to see.

https://github.com/Mervill/space-station-14/blob/80ea10b8ca5e3e5b72ed8c899eb2219b19af2b82/Content.Server/Power/EntitySystems/ChargerSystem.cs#L33-L45

```csharp
if (comp.Status == CellChargerStatus.Empty || comp.Status == CellChargerStatus.Charged || !slot.HasItem)
    continue;
```

Unless you mean a different method than this in which case please advise.

Reply:
what he means is adding a component like `ActiveChargerComponent` to the chargers with items in them, so that you literally are only iterating over active chargers--otherwise this update loop is running and checking that stuff for every charger on the map every tick